sql/src/main/java/org/apache/druid/sql/calcite/planner/CalcitePlanner.java

```diff
@@ -65,9 +61,9 @@
 import org.apache.calcite.util.Pair;
 
 import javax.annotation.Nullable;
-
 import java.io.Reader;
 import java.util.List;
+import java.util.Objects;
 import java.util.Properties;
 
 /**
```

Comment:
update the javadoc to indicate that (i assume) this has now been copied from 1.35 instead of 1.21, or if it hasn't been freshly copied we probably need to indicate what we have changed here

Reply:
Done